nl.flyingpie.wtq.yml

```diff
@@ -0,0 +1,49 @@
+id: nl.flyingpie.wtq
+runtime: org.gnome.Platform
+runtime-version: "48"
+sdk: org.gnome.Sdk
+
+command: wtq
+
+finish-args:
+  - --own-name=nl.flyingpie.wtq.svc             # Exposes DBus service for the KWin script to talk to
```

Comment:
this is not necessary, it's granted by default based on the application id.

Reply:
Done.